app/views/registered_applications/show.html.erb

```diff
@@ -2,17 +2,24 @@
 <h5>@ <%= @app.url %></h4>
 
 <h3>Events</h3>
-<div class="col-md-8">
-<table class="table">
-  <tr class="row"> 
-    <th>Name</th>
-    <th>Count</th>
-  </tr>
-<% @events.each do |event, count| %>
-    <tr class="row">
-        <td><%= event %></td>
-        <td><%= count.length %></td>
-    </tr>
-<% end %>
-</table>
+<div class="container">
+  <div class="row">
+      <%= pie_chart @app.events.group(:eventname).count %>  
+  <div class="row">
+      <%= line_chart @app.events.group_by_day(:created_at).count %>
+  </div>
+  <div class="row">
+    <table class="table">
+      <tr class="row"> 
+        <th>Name</th>
+        <th>Count</th>
+      </tr>
+      <% @events.each do |event, count| %>
+      <tr class="row">
+          <td><%= event %></td>
+          <td><%= count.length %></td>
```

Comment:
Same. I probably should have used value or something after the fact. Used count initially thinking it'd give me back the count of the keys that were grouped. Instead it gave me back an array of the key information (event info), so I had to count the length of the hash containing the arrays. Good call though for future reference.